public/index.html

```diff
@@ -58,6 +58,15 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-8" id="plot-column">
+                <div id="neighbourplot-options" class="row hidden">
```

Comment:
Is this still a pull-down? It occurs to me that we could just make it a button. I would also rename it to "scatterplot view" or something like that, a bit friendlier to users. I can't find an exact example of what I mean right now but something along the lines of the top row of this image: https://d3ui957tjb5bqd.cloudfront.net/images/screenshots/products/15/152/152410/creativemarket-f.jpg